packages/vscode-extension/src/types/ActionResult.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+
+export default interface ActionResult {
```

Comment:
This one should be a declaration file (.d.ts) as well, since you're defining the object that is coming from the javascript layer